htdocs/comm/card.php

```diff
@@ -64,6 +64,9 @@
 if (isModEnabled('intervention')) {
 	require_once DOL_DOCUMENT_ROOT.'/fichinter/class/fichinter.class.php';
 }
+if (isModEnabled('project')) {
+	require_once DOL_DOCUMENT_ROOT . '/projet/class/project.class.php';
```

Comment:
Remove the condition because, later, you use the object line 317 without condition.